pom.xml

```diff
@@ -175,6 +176,12 @@
         <configuration>
           <parallel>methods</parallel>
           <threadCount>10</threadCount>
+          <properties>
+            <property>
+              <name>surefire.testng.verbose</name>
+              <value>${verbosity}</value>
```

Comment:
This is for compatibility with the default value, and sake of simplicity.  One can just clone and set mvn -Dverbosity=10 test, wouldn't you agree?